tests/test_optimizely_config.py

```diff
@@ -616,7 +616,53 @@ def setUp(self):
                     'experiments_map': {
 
                     },
-                    'delivery_rules': [],
+                    'delivery_rules': [
+                        {
+                            'id': '211127',
+                            'key': '211127',
+                            'variations_map': {
+                                '211129': {
+                                    'id': '211129',
+                                    'key': '211129',
+                                    'feature_enabled': True,
+                                    'variables_map': {}
+                                },
+                                '211229': {
+                                    'id': '211229',
+                                    'key': '211229',
+                                    'feature_enabled': False,
+                                    'variables_map': {}
+                                }
+                            },
+                            'audiences': ''
```

Comment:
Do we have test cases to validate "audiences" in delivery rules (rollouts) are translated into OptimizelyExperiment? Otherwise, you can add valid audiences here.

Reply:
Will add test cases to verify OptimizelyExperiment in delivery Rules.